binary-codec/types/hash192.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// Hash192 struct represents a 192-bit hash.
+type Hash192 struct {
+	hashI
```

Comment:
What is `hashI` ?

Reply:
Is an interface that defines which methods should a  `HashX` type have